types/node/worker_threads.d.ts

```diff
@@ -54,6 +55,7 @@ declare module "worker_threads" {
     }
 
     interface WorkerOptions {
+        env?: NodeJS.ProcessEnv | typeof SHARE_ENV;
```

Comment:
`typeof` shouldn't be needed here.

Reply:
I believe this is correct as is, if I remove `typeof` I get an exception: `'SHARE_ENV' refers to a value, but is being used as a type here.`

Reply:
doc reference: http://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-7.html#unique-symbol

Reply:
Ah yea I misread, can you add a test for your additions tho? ❤️

Reply:
No problem! I'll have a look.